spec/javascripts/health-care/components/AddressState.spec.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+import AddressState from
+    '../../../../_health-care/_js/components/personal-information/AddressState';
+
+describe('<AddressState>', () => {
+  let component = null;
+
+  beforeEach(() => {
+    component = ReactTestUtils.renderIntoDocument(
+      <AddressState state={{ state: 'CA' }} onUserInput={(_unused) => {}}/>
```

Comment:
Let's rename the API to be a bit more semantically consistent:
  https://github.com/department-of-veterans-affairs/vets-website/blob/master/_health-care/_js/components/form-elements/ErrorableTextInput.jsx#L13

I used the `state` and `onUserInput` names originally cause I didn't understand the API design.  These attributes are basically the function input, so we should try to name them to semantically match the timbre of the component, which in this case is effectively a pretty wrapper around <select> so `value` and `onValueChange` matches the DOM analogous DOM concepts.

@akainic tagging so you can see my thinking.

Push back if this doesn't seem right to either of you.